src/tareas.ts

```diff
@@ -0,0 +1,76 @@
+interface Tasks {
+  id?: string;
+  title: string;
+  description: string;
+  complete?: boolean;
+}
+
+class List {
+  tasks: Tasks[] = [];
+
+  addTask(id: string, title: string, description: string): void {
+    const newTask: Tasks = {
+      id,
+      title,
+      description,
+      complete: false,
+    };
+
+    this.tasks.push(newTask);
+    `Nueva tarea agragada: ${title}`;
+  }
+
+  edit(id: string, task: Tasks, newTitle: string, newDescription: string) {
```

Comment:
para seguir el patron
```suggestion
  editTask(id: string, task: Task, newTitle: string, newDescription: string) {
```

Reply:
cuando me digas: si TaskS (plural) o Task (singular)

Reply:
singular